docs/index.md

```diff
@@ -34,6 +34,19 @@ into your `PATH`:
 - [Linux - x86_64](https://github.com/docker/machine/releases/download/v0.1.0/docker-machine_linux-amd64)
 - [Linux - i386](https://github.com/docker/machine/releases/download/v0.1.0/docker-machine_linux-386)
 
+### OSX and Linux
+
+To install on OSX or Linux, download the proper binary to somewhere in your
+`PATH` (e.g. `/usr/local/bin`) and make it executable.  For instance, to install on
+most OSX machines these commands should suffice:
+
+```
+$ curl https://github.com/docker/machine/releases/download/v0.1.0/docker-machine_darwin-amd64 > /usr/local/bin/docker-machine
```

Comment:
AFAIK we haven't moved to become compatible with the compose-style "uname -s" curl yet, but I want to move there eventually with this section.